modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -37,7 +37,13 @@ The {ai-full} supports installing {product-title} on premises in a connected env
 * Highly available {product-title} or {sno} (SNO)
 
 * {product-title} on bare metal, Nutanix, or vSphere with full platform integration, or other virtualization platforms without integration
-* Optionally {VirtProductName} and {rh-storage} (formerly OpenShift Container Storage)
+
+* Optionally, {VirtProductName}, {rh-storage}, multicluster engine, and {lvms-first}
++
+[NOTE]
+====
+Currently, {VirtProductName} and {lvms} are not supported on {ibm-z-name} (`s390x`) architecture.
```

Comment:
The order on the panel is: OpenShift Virtualization, multicluster engine, Logical Volume Manager (LVM) Storage and OpenShift Data Foundation. Not sure if this matters but I thought worth to be mentioned.